### PR TITLE
server/tso: fix the panic issue when getTS in the follower

### PR DESCRIPTION
**Pull request by nolouch**
Signed-off-by: nolouch <nolouch@gmail.com>

### What problem does this PR solve? 
Details:

image

### What is changed and how it works?
fix the nil pointer.

### Check List 

Tests 

 - Unit test

### CODE REVIEW

File: server/tso/tso.go [hunk 1]
```diff
@@ -265,9 +265,13 @@ func (t *TimestampOracle) GetRespTS(count uint32) (pdpb.Timestamp, error) {
 		return resp, errors.New("tso count should be positive")
 	}
 
+	failpoint.Inject("skipRetryGetTS", func() {
```
Comment (lhy1024) [inline server/tso/tso.go]: can we add `maxRetryCount` to `TimestampOracle` to avoid using failpoint?

Comment (nolouch) [reply server/tso/tso.go]: Let's keep this variable unchanged inside bug fix firstly.

**Pull request merged**